tests/openvino/test_modeling.py

```diff
@@ -66,6 +70,21 @@
 SEED = 42
 
 
+class ORTModelIntegrationTest(unittest.TestCase):
```

Comment:
why the test is called `ORTModelIntegrationTest`?

Reply:
Good catch, I'll rename it !